### PR TITLE
documented last-change-trigger-time

### PR DESCRIPTION
**Pull request by kundan2707**
Fixes: #38000

### CODE REVIEW

**Review by Princesso — COMMENTED** [review]
/assign @sftim

**Comment by sftim** [comment]
Anyone is welcome to review this.
/unassign

however, I will take a look

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1294,6 +1294,18 @@ has been truncated to 1000.
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: 2023-07-20T04:45:21Z`
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1301]: For clarity:
```suggestion
Example: `endpoints.kubernetes.io/last-change-trigger-time: "2023-07-20T04:45:21Z"`
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -1294,6 +1294,18 @@ has been truncated to 1000.
...
+
+Used on: Endpoints
+
+This annotation set to an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object that
+represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z'). This is timestamp
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1306]: (nit) We [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -1294,6 +1294,18 @@ has been truncated to 1000.
...
+of the last change in some Pod or Service object, that triggered the endpoints object change.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1307]:
```suggestion
of the last change in some Pod or Service object, that triggered the change to the Endpoints object.
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
I'd welcome further fixes

**Comment by divya-mohan0209** [comment]
@kundan2707 : Please could you advise if you have the bandwidth to make the above changes suggested by Tim?

**Comment by sftim** [comment]
To be clear: I'd _welcome_ more fixes, but I don't object to merging this as-is.

**Comment by reylejano** [comment]
Approving as-is
/approve

**Comment by reylejano** [comment]
Follow-up PR https://github.com/kubernetes/website/pull/45969

**Pull request merged**